pkg/operator/ceph/csi/util.go

```diff
@@ -132,13 +144,25 @@ func getToleration(provisioner bool) []corev1.Toleration {
 	return tolerations
 }
 
-func getNodeAffinity(provisioner bool) *corev1.NodeAffinity {
+func getNodeAffinity(clientset kubernetes.Interface, namespace string, provisioner bool) *corev1.NodeAffinity {
 	// Add NodeAffinity if any
 	nodeAffinity := ""
-	if provisioner {
-		nodeAffinity = os.Getenv(provisionerNodeAffinityEnv)
-	} else {
-		nodeAffinity = os.Getenv(pluginNodeAffinityEnv)
+	// TODO: export config map name
+	rookOperatorConfig, err := k8sutil.GetConfigMap(clientset, "rook-operator-config", namespace)
```

Comment:
we should not enforce the user to create configmap with the same name as `rook-operator-config`

Reply:
my vote is for `rook-ceph-operator-config`